### PR TITLE
[RB] Fix bmi DB error

### LINKED ISSUE #8578: [instruments] Error on saving LINST data
**Describe the bug**

Screenshot 2023-04-06 at 1 42 38 PM

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):

**What did you expect to happen?**

**Browser Environment (please complete the following information):**
- OS: [e.g. Plan9, HURD, Oberon, etc]
- Browser [e.g. chrome, firefox] (note that only these two browsers are supported)

**Server Environment (if known):**
_Note: We only support the most recent release of LORIS._
- LORIS Version: [e.g. 22.0.0]
- Linux distribution and Version: [e.g. Ubuntu 16.04, CentOS 7]
- MySQL/MariaDB Version: [e.g. MySQL 5.7, MariaDB 10.3]

**Additional context**

### PR DESCRIPTION
**Pull request by laemtl**
An inconsistency between the linst file and the sql schema in the enum field unit_classification trigger a 500 error. Fixed by updating the enum with the missing value in the SQL schema.

Fixes #8578

### CODE REVIEW

**Comment by laemtl** [comment]
@zaliqarosli I apply the changes in the test VM to help your tests.

File: raisinbread/instruments/bmi.linst [hunk 1]
```diff
@@ -1,23 +1,24 @@
+table{@}bmi
```
Comment (zaliqarosli) [inline raisinbread/instruments/bmi.linst:1]: i think this element now lives in bmi.meta so doesn't need to be here as well

File: raisinbread/instruments/instrument_sql/bmi.sql [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
-
-                            `Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+`UserID` varchar(255) default NULL,
+`Examiner` varchar(255) default NULL,
+`Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
 `Date_taken` date default NULL,
 `Candidate_Age` varchar(255) default NULL,
 `Window_Difference` varchar(255) default NULL,
-`unit_classification` enum('metric','standard') default NULL,
+`unit_classification` enum('metric','imperial','standard','not_answered') default NULL,
```
Comment (zaliqarosli) [inline raisinbread/instruments/instrument_sql/bmi.sql:9]: i think the options should just be metric and imperial so its just the 'standard' in the SQL that needs to be changed to 'imperial'

File: raisinbread/instruments/bmi.linst [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
 static{@}Candidate_Age{@}Candidate Age (Months)
 static{@}Window_Difference{@}Window Difference (+/- Days)
 select{@}Examiner{@}Examiner{@}NULL=>''
 static{@}{@}Note: Please enter measurements in Imperial OR Metric units
-select{@}unit_classification{@}Choose unit of measurement{@}NULL=>''{-}'metric'=>'Metric'{-}'imperial'=>'Imperial'
+select{@}unit_classification{@}Choose unit of measurement{@}NULL=>''{-}'metric'=>'Metric'{-}'imperial'=>'Imperial'{-}'standard'=>'Standard'{-}'not_answered'=>'Not Answered'
 page{@}{@}Imperial units
 static{@}{@}Imperial units
-numeric{@}height_feet{@}Your height (feet) :
+numeric{@}height_feet{@}Your height (feet) :{@}undefined{@}undefined
```
Comment (zaliqarosli) [inline raisinbread/instruments/bmi.linst:11]: i think this 'undefined' might be coming from a bug in the instrument builder? unless this is the max/min in which case i think leaving it blank should be better. not sure if 'undefined' case is being handled in the NDB_BVL_Instrument_LINST class

**Review by zaliqarosli — CHANGES_REQUESTED** [review]

**Comment by laemtl** [comment]
@zaliqarosli Ready for second review

**Review by zaliqarosli — CHANGES_REQUESTED** [review]
i think changes to `raisinbread/instruments/bmi.linst` can be removed completely cz its just a permissions change which i think is unnecessary

**Comment by laemtl** [comment]
Yes, the 400 should fo away once #8651 is merged. I fixed the perm change, thanks for the hint :)

**Comment by laemtl** [comment]
New error fixed in #8681

**Review by zaliqarosli — APPROVED** [review]

**Pull request merged**